tensorflow/lite/examples/android/app/README.md

```diff
@@ -17,9 +17,9 @@ for our external and internal code to merge.
 1. Follow the [Bazel steps for the TF Demo App](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#bazel):
 
   1. [Install Bazel and Android Prerequisites](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#install-bazel-and-android-prerequisites).
-     It's easiest with Android Studio.
+     It's easy with Android Studio.
```

Comment:
I think the superlative is better here as Android Studio is best equipped to handle this and hence the 'easiest'

Reply:
sure changing it.